### ISSUE TITLE
General Review Feedback

### ISSUE THREAD

**Issue by mrporcles** [mbahvw/kit-katacoda-scenarios#10]
Looks good. I think the level is just right for all 3 scenarios. Just a few notes/comments:

### Intro Scenario
- [ ] **Step 1** - The env for the scenario took a few mins to initialise for me so by the time you get to the kubectl cluster-info it fails (It seems to reuse the initial environment for the subsequent scenarios). It also doesn't seem to put a kubeconfig in place so I had to do that manually.

- [ ] **Step 2** - Should the last instruction have an execution (for kubectl get)?

- [ ] **Step 6** - Number of CrashLoopBackOff from katacoda-cloud-provider when listing all pods

- [ ] **Step 9** - I would say "Number of Replicas" instead of "Amount of Replicas"

### Intermediate Scenario

- [ ] **Step 3** - Half way down you have "Type or click the below command to show the context:"
You don't seem to mention click for commands in any of the previous steps or scenarios. I would just say "Type the below command..." for consistency.

- [ ] **Step 4** - To switch between contexts, type kubectx CONTEXT NAME - I would change this to "To switch between contexts, you can now type kubectx CONTEXT NAME as it implies you should type the command as shown even though you don't have an execution. I would add another execution of the kubectx command to show it has changed context to dev-context before going on to changing back to kubernetes-admin.

- [ ] **Step 5** - You've mentioned click for the command again and in Step 6. I guess decide if you want to keep these as is. I won't comment on further ones.

- [ ] **Step 6** - This yields the same thing as the --generator=run/v1 above. I would reword this to "This yields the same result as the --generator=run/v1 configuration option above"

- [ ] **Step 10** - Spelling error - famility should be familiarity. "Pre-requisite: Familiarity with json and jsonpath is crucial as well as famility"

- [ ]  **Step 11** - Output on command line lists CPU as 4 for master not 2 as shown in the instructions.

- [ ] **Step 15** - I would change this "Here is the full command:" to "Here is the full command you would use:"

- [ ] **Step 17** - The nginx manifest deployment fails as you already have a deployment with the same name from a previous step. Either need to add a command to delete the previous one or change the name of the new deployment.

### Advanced Scenario

- [ ] **Step 1** - The link to the intermediate scenario has a name mismatch. You have actually named it "Kubectl intermediate" scenario not "intermediate kubectl".

- [ ] **Step 4** - Perhaps add a line between the 2 who-can commands. I would also add a line between the 2 commands for pod-dive. The last command should be "kubectl config get-contexts". Not "kubectl get contexts".

- [ ] **Step 6** - You could make cat /tmp/welcome.txt an executable command. I wouldn't make the last command executable as it leaves the cli shelled to the pod and the next commands will fail unless you exit.

- [ ] **Step 7** - I would make "ls /tmp/krew-install" executable. For copying from pod to node there is no welcome.txt on the pod so will fail.

- [ ] **Step 10** - I would put the uncordon node01 command on the next line.

- [ ] **Step 11** - I would add a line between the 2 commands at the bottom.

**Comment by mbahvw** [comment]
Thanks Paul for this great feedback, we truly appreciate it.
